src/ol_concourse/pipelines/libraries/scripts/open-api-clients-publish-node.sh

```diff
@@ -2,4 +2,4 @@
 # publish the npm package to npmjs.org
 
 yarn build
```

Comment:
To fix `/bin/sh: 1: tsc: not found`:

```suggestion
yarn install --immutable
yarn build
```